src/sdk/model/company/__json__/cloud-tenant-bill-json.ts

```diff
@@ -0,0 +1,11 @@
+import { CurrencyCodeJson } from './currency-code-json';
+
+export interface CloudTenantBillJson {
+  total: number;
+  year: number;
+  month: number;
+  currency_code: CurrencyCodeJson;
```

Comment:
why does the api not just place the enum value in this property directly vs. having another object as a layer of indirection?

Reply:
I just used the auto generated ones. I can change it.

Reply:
or do you mean java side @cfsnyder ?

Reply:
yeah i meant java side